presto-main/src/main/java/com/facebook/presto/execution/TaskStatus.java

```diff
@@ -70,7 +71,8 @@
 
     @JsonCreator
     public TaskStatus(
-            @JsonProperty("taskInstanceId") String taskInstanceId,
+            @JsonProperty("taskInstanceIdLsb") long taskInstanceIdLsb,
+            @JsonProperty("taskInstanceIdMsb") long taskInstanceIdMsb,
```

Comment:
I think we should create a class to represent taskInstanceId  that  holds long taskInstanceIdLsb and long taskInstanceIdMsb. It may increase the overhead on JSON.

Reply:
done